h/search/config.py

```diff
@@ -236,7 +317,7 @@ def delete_index(client, index_name):
 
     try:
         client.conn.indices.delete(index=index_name)
-    except NotFoundError:
+    except elasticsearch1.exceptions.NotFoundError:
```

Comment:
I don't know if we'll encounter this with ES 6 as well. Let's keep an eye open when we run reindexing operations.